test/python/washita/LW_Test.py

```diff
@@ -561,4 +565,21 @@
 # Run Simulation
 # -----------------------------------------------------------------------------
 
+# Corrupt the indicator file and run check_nans()
+corrupted_indicator = read_pfb(os.path.join(dir_name, "IndicatorFile_Gleeson.50z.pfb"))
+corrupted_indicator[0, 0, 0] = np.nan
+write_pfb(os.path.join(dir_name, "corrupted_indicator.pfb"), corrupted_indicator)
+LW_Test.Geom.indi_input.FileName = "corrupted_indicator.pfb"
+try:
+    LW_Test.check_nans(working_directory=dir_name)
+    raise AssertionError("Expected ValueError, but none was raised.")
+except ValueError as e:
+    assert "corrupted_indicator.pfb" in str(e)
+    print("ValueError was raised as expected for corrupted indicator file.")
```

Comment:
We tend to not print a lot of successful messages to make the successful test runs not very verbose.